core/lib/sync/task.ts

```diff
@@ -118,7 +118,7 @@ function _internalExecute (resolve, reject, task: Task, currentToken, currentCar
             Vue.prototype.$bus.$emit('modal-show', 'modal-signup')
           }
         }
-        if (!task.silent && (jsonResponse.result && jsonResponse.result.code !== 'ENOTFOUND' && !silentMode)) {
+        if (!task.silent && (typeof jsonResponse.result === 'string' && jsonResponse.result.code !== 'ENOTFOUND' && !silentMode)) {
```

Comment:
`(typeof jsonResponse.result === 'string' && jsonResponse.result.code !== 'ENOTFOUND'`
if it is a string, then it doesn't have code

Reply:
you're right. I've changed that